src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -141,6 +142,11 @@ public Patch? SelectedPatch
     /// </summary>
     public Action<PatchMapDrawer>? OnSelectedPatchChanged { get; set; }
 
+    /// <summary>
+    ///   Player species ID for player population indicator (dots on patch map)
+    /// </summary>
+    public uint PlayerSpeciesID { get; set; }
```

Comment:
I think this could default to -1 (or uint.MaxValue if C# complains) so that this doesn't accidentally match anything if there's a bug elsewhere in the code that causes a species with ID = 0 to be created.